safe-global/safe-wallet-web
# Feat: decode approve hash

## What it solves

Resolves [SW-352](https://www.notion.so/safe-global/Decode-nested-approveHash-transactions-1228180fe573800ebc6acd3f8e54035c)

## How this PR fixes it
Fetches and displays transaction details of the Safe transaction behind the txHash.

## How to test it
- Sign a transaction with a nested Safe in a Safe with threshold > 1
- Open the on-chain signature tx in the parent Safe
- Observe that the nested transaction is displayed

## Screenshots
![Screenshot 2024-10-25 at 14 53 03](https://github.com/user-attachments/assets/cfb41ed6-8ce9-4eba-8e09-d877a3f42b83)

## Checklist
